### ISSUE TITLE
Log file and line

### ISSUE THREAD

**Issue by stgleb** [x-cray/logrus-prefixed-formatter#11]
Is therere any possibility to log file and line like

```
Info = log.New(infoHandle,
        "INFO: ",
        log.Ldate|log.Ltime|log.Lshortfile)
```

In standard package log ?

**Comment by x-cray** [comment]
@stgleb No, currently it's not possible with the formatter. Apparently, there's a [feature request](https://github.com/sirupsen/logrus/issues/63) in logrus. And I'd like to monitor that issue prior to implementing of this feature in formatter. PR is appreciated, however.
